infrastructure/images/moduletest-reset-jobs/Chart.yaml

```diff
@@ -0,0 +1,3 @@
+apiVersion: v2
+name: moduletest-file-reset
+version: 0.0.1
```

Comment:
Unless you plan to publish the chart somewhere, the version doesn't matter. I usually set it to 0.0.0 but maybe a comment is also in order.

Reply:
fixed